model/cloud.go

```diff
@@ -124,6 +124,10 @@ type ValidateBusinessEmailResponse struct {
 	IsValid bool `json:"is_valid"`
 }
 
+type SubscriptionExpandStats struct {
+	IsExpandable bool `json:"is_expandable"`
```

Comment:
Nit: I think we want all the code in in webapp, server and enterprise to be `ExpandStatus` instead of `ExpandStats`. I'd expect "stats" to be things like numerical counts of things, and customer-web-server uses "status"